### PR TITLE
Breakpoint dependencies and Import path

### PR DESCRIPTION
**Pull request by ingedmundo**
- Clears unnecessary dependencies for breakpoint
- Changes the import path for the bower package
- Refactors import order on main file

### CODE REVIEW

File: assets/stylesheets/base.scss [hunk 1]
```diff
@@ -13,3 +13,8 @@ p {
   line-height: 1.25em;
   margin: 0 auto;
 }
+
```
Comment (mumoc) [inline assets/stylesheets/base.scss]: Food for thought: Why did you decide on these values?

File: assets/stylesheets/main.scss [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+/* Import fonts */
+@import 'fonts/lato';
+@import 'fonts/typography';
+//
+///* Import layout files */
 @import 'layouts/content';
 @import 'layouts/footer';
 @import 'layouts/header';
-@import 'layouts/grid';
 @import 'layouts/grid2';
```
Comment (mumoc) [inline assets/stylesheets/main.scss]: Just a quick question here: Why are you keeping reference to grid2 if you've deleted grid? I recommend you to rename grid2 to grid

File: assets/stylesheets/base.scss (hunk 1 above)
Comment (ingedmundo) [inline assets/stylesheets/base.scss]: This is comming from a merge from master, I think @zalapeach put those values, maybe he can answer your question.

But quick answer is: In the styleguide they appear to have double or 1.5 line height.

File: assets/stylesheets/main.scss (hunk 2 above)
Comment (ingedmundo) [inline assets/stylesheets/main.scss]: Grid2 and Grid are still in 'beta' but, yes... I think it makes more sense to keep just grid as the grid definition file... I mean, It makes more sense that grid2... (where is grid1? dhu?!)

**Comment by mumoc** [comment]
Rename the file grid2 to grid and will LGTM.

On the breakpoints we could discuss about it chat ^^. Those values are fine; just wanna make sure there were a real process defining them ^^

File: assets/stylesheets/main.scss (hunk 2 above)
Comment (efigarolam) [inline assets/stylesheets/main.scss]: +1

**Comment by vicmaster** [comment]
I agree with @mumoc, beside that LGTM

**Pull request merged**